src/esp/gfx/RenderTarget.cpp

```diff
@@ -337,8 +340,8 @@ RenderTarget::RenderTarget(const Mn::Vector2i& size,
                                             depthShader,
                                             flags)) {}
 
-void RenderTarget::renderEnter() {
-  pimpl_->renderEnter();
+void RenderTarget::renderEnter(sensor::CameraSensor& cameraSensor) {
```

Comment:
I meant in the constructor itself.  This works but it kinda invalidates the context manager on the python side.  If there is some reason I am missing for why we can't have it be in the constructor, the we should get rid of the context manager in python

Reply:
🤔  Looks like there are a number of tests which create RenderTargets for direct consumption with a RenderCamera and have no associated CameraSensor. I guess an optional CameraSensor member could be used, but we'll need a default color anyway in that case.

Reply:
Yeah, optional with a default color is fine by me.  Adds an if, but it's a pretty tame one

Reply:
Using your suggested constructor approach, agreed it's cleaner. Since RenderTarget is constructed in VisualSensor I elevated the clearColor to the VisualSensorSpec.